controllers/auth.js

```diff
@@ -0,0 +1,58 @@
+const User = require('../models/User')
+const bcrypt = require('bcrypt')
+
+const create = async ( req, res ) => {
+    try {
+      
+      let {password} = req.body
```

Comment:
Should we also have a password vs. password 2 check

Reply:
No, Tosin said password matching is the front end's responsibility